src/mscorlib/shared/System/Diagnostics/Tracing/TraceLogging/TraceLoggingEventSource.cs

```diff
@@ -5,13 +5,13 @@
 // This program uses code hyperlinks available as part of the HyperAddin Visual Studio plug-in.
 // It is available from http://www.codeplex.com/hyperAddin 
 
-#if !PLATFORM_UNIX
+#if PLATFORM_WINDOWS
```

Comment:
I am not sure whether the Windows builds defines PLATFORM_WINDOWS. May need to add it.